networkx/algorithms/approximation/treewidth.py

```diff
@@ -57,6 +57,18 @@ def treewidth_min_degree(G):
     -------
     Treewidth decomposition : (int, Graph) tuple
           2-tuple with treewidth and the corresponding decomposed tree.
+
+    Examples
+    --------
+    >>> G = nx.Graph()
+    >>> edges = [(0, 1),(0, 3),(1, 2),(1, 3),(1, 4)]
+    >>> G.add_edges_from(edges)
+    >>> treewidth = treewidth_min_degree(G)
+    >>> treewidth[0]
+    2
+    >>> type(treewidth[1])
+    <class 'networkx.classes.graph.Graph'>
```

Comment:
The `type` of the returned tree is not particularly interesting - it might be better to show the decomposition:

```suggestion
    >>> width, tree = nx.approximation.treewidth_min_degree(G)
    >>> width
    2
    >>> print(tree)
    Graph with 3 nodes and 2 edges
    >>> tree.edges()
    EdgeView([(frozenset({0, 1, 3}), frozenset({1, 4})), (frozenset({0, 1, 3}), frozenset({1, 2}))])
```

Note also that treewidth_min_degree was never actually imported in the docstring example.